docker/Dockerfile

```diff
@@ -31,10 +24,21 @@ RUN apt-get update && \
         gcc \
         git git-core \
         mariadb-client \
+        pkg-config \
         libmariadbclient-dev-compat \
-        unzip curl wget sudo ssh \
-        && \
-    apt-get clean && \
+        unzip curl wget sudo ssh
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_20.x | bash - && \
+    apt-get install -y nodejs
+
+# Install yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends yarn
```

Comment:
I think we shouldn't compile the JS while the image is created, shouldn't it be something done before?

Reply:
Yes, I was using `poetry install` to build the image. I will update it to use the package instead.

I will update the other PR that this depends on and rebase this one.